src/wayland-egldisplay.c

```diff
@@ -454,6 +455,11 @@ registry_handle_global(void *data,
                                                    name,
                                                    &wp_presentation_interface,
                                                    version);
+    } else if (strcmp(interface, "wp_linux_drm_syncobj_manager_v1") == 0) {
+        display->wlDrmSyncobj = wl_registry_bind(registry,
```

Comment:
Should we check if `EGL_ANDROID_native_fence_sync` is supported before using explicit sync?